src/use-cases/commands/ModifierUnComite.test.ts

```diff
@@ -87,39 +87,39 @@ describe('modifier un comité', () => {
   it.each([
     {
       date,
-      dateDeModification: date,
+      dateDeModification: epochTime,
       expectedFailure: 'frequenceInvalide',
       frequence: 'frequenceInvalide',
       intention: 'd’une fréquence invalide',
       type: typeValide,
     },
     {
       date,
-      dateDeModification: date,
+      dateDeModification: epochTime,
       expectedFailure: 'typeInvalide',
       frequence: frequenceValide,
       intention: 'd’un type invalide',
       type: 'typeInvalide',
     },
     {
       date: 'dateDuComiteInvalide',
-      dateDeModification: date,
+      dateDeModification: epochTime,
       expectedFailure: 'dateDuComiteInvalide',
       frequence: frequenceValide,
       intention: 'de la date invalide',
       type: typeValide,
     },
     {
       date,
-      dateDeModification: 'dateDeModificationInvalide',
+      dateDeModification: invalidDate,
       expectedFailure: 'dateDeModificationInvalide',
       frequence: frequenceValide,
       intention: 'de la date de modification invalide',
       type: typeValide,
     },
     {
-      date: '2024-01-01',
-      dateDeModification: '2024-01-02',
+      date: '1970-01-01',
```

Comment:
on peut mettre directe date ?!
```suggestion
      date,
```